covidanalyzer.py

```diff
@@ -0,0 +1,38 @@
+import argparse
+from allcalculations import *
+
+# Create the parser
+parser = argparse.ArgumentParser()
+# Add an argument
+parser.add_argument('-a', type=str)
+parser.add_argument('-b', type=str)
+parser.add_argument('-c', type=str)
+# Parse the argument
+args = parser.parse_args()
+
+
+if args.a is not None:
```

Comment:
Look up truthy and falsy values in python.

Reply:
ok

Reply:
corrected the code with the truthy and falsy syntax